src/components/Dialog/Dialog.visual.tsx

```diff
@@ -3,36 +3,46 @@ import { visualize, story, snap } from 'storybook-snapper';
 import { Dialog } from './';
 import { Text, TYPOGRAPHY } from '../Text';
 import { Button, PRIORITY } from '../Button';
+import { setDarkPalette } from '../../test/visualTestUtils';
+
+const DialogWithContent = props => (
+  <Dialog isOpen wiredToSiteColors={props.wired}>
+    <div className="content" style={{ textAlign: 'center' }}>
+      <Text typography={TYPOGRAPHY.largeTitle}>Are You Sure?</Text>
+      <div
+        className="text-container"
+        style={{ marginTop: '24px', marginBottom: '36px' }}
+      >
+        <Text typography={TYPOGRAPHY.listText} tagName="div">
+          <div>Do you really want to delete the selected files?</div>
+          <div>Once removed, cannot be undone.</div>
+        </Text>
+      </div>
+      <Button
+        upgrade
+        priority={PRIORITY.basicSecondary}
+        style={{ marginLeft: '10px' }}
+      >
+        SECONDARY
+      </Button>
+      <Button upgrade style={{ marginLeft: '10px' }}>
+        PRIMARY
+      </Button>
+    </div>
+  </Dialog>
+);
 
 visualize('Dialog', () => {
   story('simple', () => {
     snap('default props', <Dialog isOpen />);
-    snap(
-      'Dialog with some content',
-      <Dialog isOpen>
-        <div className="content" style={{ textAlign: 'center' }}>
-          <Text typography={TYPOGRAPHY.largeTitle}>Are You Sure?</Text>
-          <div
-            className="text-container"
-            style={{ marginTop: '24px', marginBottom: '36px' }}
-          >
-            <Text typography={TYPOGRAPHY.listText} tagName="div">
-              <div>Do you really want to delete the selected files?</div>
-              <div>Once removed, cannot be undone.</div>
-            </Text>
-          </div>
-          <Button
-            upgrade
-            priority={PRIORITY.basicSecondary}
-            style={{ marginLeft: '10px' }}
-          >
-            SECONDARY
-          </Button>
-          <Button upgrade style={{ marginLeft: '10px' }}>
-            PRIMARY
-          </Button>
-        </div>
-      </Dialog>,
-    );
+    snap('Dialog with some content', <DialogWithContent />);
+  });
+
+  story('Wired to palette', () => {
+    snap('Dialog with some content', () => {
+      setDarkPalette();
```

Comment:
This will cause a side effect. every test that comes afterward will stay in the dark palette.

Reply:
I was hoping the tests would run in parallel, so that it won't affect the rest of the tests.
I think the batch runs in the same window 😞